src/IdentityManager/DataService/IdentityServices/PersistedGrantStore.cs

```diff
@@ -0,0 +1,128 @@
+﻿using AutoMapper;
+using IdentityManager.DataService.Mappers;
+using IdentityServer4.Extensions;
+using IdentityServer4.Models;
+using IdentityServer4.Stores;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Travely.IdentityManager.Repository.Abstractions;
+using entityModel = Travely.IdentityManager.Repository.Abstractions.Entities;
+
+namespace IdentityManager.DataService.IdentityServices
+{
+    public class PersistedGrantStore : IPersistedGrantStore
+    {
+        IPersistGrantRepository _persistGrantRepository;
+        IUnitOfWork _unitOfWork;
+        ILogger<PersistedGrantStore> _logger;
+        IMapper _mapper;
+        public PersistedGrantStore(ILogger<PersistedGrantStore> logger, IPersistGrantRepository persistGrantRepository, IUnitOfWork unitOfWork, IMapper mapper)
+        {
+            _persistGrantRepository = persistGrantRepository;
+            _unitOfWork = unitOfWork;
+            _logger = logger;
+            _mapper = mapper;
+        }
+
+        public async Task<IEnumerable<PersistedGrant>> GetAllAsync(PersistedGrantFilter filter)
+        {
+            filter.Validate();
+
+            var persistedGrants = await Filter(_persistGrantRepository.GetAll(), filter).ToArrayAsync();
+
+            var result = _mapper.Map<List<PersistedGrant>>(persistedGrants);
```

Comment:
Still not ProjectTo. Use this extension method on IQueryable 
https://github.com/AutoMapper/AutoMapper/blob/565c9634e2a0b487c9b25bca20f67a43e0ee06a3/src/AutoMapper/QueryableExtensions/Extensions.cs#L36